plugins/webhooks/index.js

```diff
@@ -399,19 +447,47 @@ async function createPREvents(options, request) {
                 prRef,
                 prNum,
                 prTitle,
-                prInfo: await eventFactory.scm.getPrInfo(scmConfig),
+                prInfo,
                 prSource,
                 baseBranch: branch
             };
 
-            if (skipMessage) {
-                eventConfig.skipMessage = skipMessage;
-            }
-
             if (b === branch) {
                 eventConfig.startFrom = '~pr';
             }
 
+            // Check if the webhook event is from a subscribed repo and
+            // set the jobs entrypoint from ~startfrom
+            // For subscribed PR event, it should be mimiced as a commit
```

Comment:
what does this mean? should this be `type: 'pr'`?

Reply:
pr event happened on the subscribed git repo. But for the subscriber it's similar to running a `commit` event